packages/workflow/README.md

```diff
@@ -0,0 +1,5 @@
+# Install
+
+```
+yarn add @kaizen/workflow
+```
```

Comment:
Don't think you need this, they should be coming from Storybook?

Reply:
I think this came from plop, we could axe that from the template if we don't need those in the future